shared/src/commonMain/kotlin/dev/johnoreilly/confetti/di/Koin.kt

```diff
@@ -9,31 +9,28 @@ import dev.johnoreilly.confetti.ConfettiRepository
 import dev.johnoreilly.confetti.TokenProvider
 import org.koin.core.context.startKoin
 import org.koin.core.module.Module
+import org.koin.core.module.dsl.singleOf
 import org.koin.dsl.KoinAppDeclaration
 import org.koin.dsl.module
 
 expect fun platformModule(): Module
 
 fun initKoin(appDeclaration: KoinAppDeclaration = {}) =
     startKoin {
-        modules(commonModule(), platformModule())
+        modules(commonModule())
         appDeclaration()
     }
 
 // called by iOS client
 fun initKoin() = initKoin() {}
 
 fun commonModule() = module {
-    single { ConfettiRepository() }
-    single { AppSettings(get()) }
-    single { ApolloClientCache() }
-    single<TokenProvider> {
-        object : TokenProvider {
-            override suspend fun token(forceRefresh: Boolean): String? {
-                return null
-            }
-        }
-    }
+    includes(platformModule())
```

Comment:
`includes` automatically flattening and removing any redundant binds from Koin - the current set-up is small so it has minimum impact, but it is nice to have.